libs/usb-drive/src/mount.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -euo pipefail
+
+usage () {
+    echo 'Usage: mount.sh <device>'
+    exit 1
+}
+
+if ! [[ $# -eq 1 ]]; then
+    usage
+fi
+
+USB_DRIVE_DEVICE_REGEX=^/dev/sd[a-z][0-9]$
+LOOP_DEVICE_REGEX=^/dev/loop[0-9]p[0-9]$
+
+if ! [[ $1 =~ $USB_DRIVE_DEVICE_REGEX || $1 =~ $LOOP_DEVICE_REGEX ]]; then
```

Comment:
this loop device regex is specific to the way that `libs/usb-mocking` is attaching a mock volume. if you don't see this library being used with `libs/usb-mocking`, you can remove. if you do see them being used together in the interim, makes sense to keep.

Reply:
I think we will probably use them together for now, but that's good to know!